mypy.ini

```diff
@@ -11,11 +11,17 @@ no_implicit_optional = True
 
 # Strict checks, see mypy --help
 warn_unused_configs = True
+# disallow_any_generics = True
 disallow_subclassing_any = True
+# disallow_untyped_calls = True
 disallow_untyped_defs = True
 disallow_incomplete_defs = True
+# check_untyped_defs = True
```

Comment:
I'm surprised---I thought we had this option enabled!

Reply:
This keeps confusing me, but `check_untyped_defs` is when mypy attempts to check the internals of functions that are not fully typed. Since we use `disallow_untyped_defs` this mostly doesn't apply to us, except for the treecache stuff which I've failed several times at typing now.